Makefile

```diff
@@ -15,6 +15,8 @@ build: fmtcheck go-lint
 generate: tools
 	# generate object tests
 	tools/tools object-generator --mode tests --in ./pkg/... --out xxgenerated_object_test.go
+	# generate GetIdentifier methods for objects
+	tools/tools object-identifier
```

Comment:
if you look at the command above, you can see the `--mode tests` - this was intended to allow the code generator to run for "objects" in different modes, like "tests" and "runtime". The new generator you built is a part of a new "runtime"-mode "object-generator".

It should support the same `--in` and `--out` as the current object-generator in tests-mode.

Reply:
You can integrate your new code into the existing object_generator code, reducing a lot of duplicate code. Directory traversal, "code is generated!"-header and co is all already done.

Ping me if you need pointers how to do that.